client/js/shout.js

```diff
@@ -374,18 +374,34 @@ $(function() {
 		thumbnails: true,
 		quit: true,
 		notifyAllMessages: false,
+		timestamp: "HH:mm"
 	}, $.cookie("settings"));
 
+	$("#timestamp").on("change", function() {
+		var self = $(this);
+		window.timestamp = self.val();
+		$(".time").each(function() {
+			var newtime = window.renderTime(self.attr("data-time"));
```

Comment:
You can directly invoke `Handlebars.helpers.tz`, no need to make a new window function.